internal/appsec/waf.go

```diff
@@ -105,7 +105,7 @@ func newWAFEventListener(handle *waf.Handle, addresses []string, appsec *appsec,
 				spanCtx := span.Context()
 				event = withSpanContext(event, spanCtx.TraceID(), spanCtx.SpanID())
 				// Keep this span due to the security event
-				span.SetTag(ext.ManualKeep, true)
+				span.SetTag(ext.ManualKeep, 5)
```

Comment:
@Julio-Guerra What do you think of this change. `5` is coming from the below table
|   Sampler   | ID |
|:-----------:|:--:|
|     Unknown | -1 |
| Default     | 0  |
| Agent Rate  | 1  |
| Remote Rate | 2  |
| Rule        | 3  |
| Manual      | 4  |
| AppSec      | 5  |

Reply:
This breaks the specification. That's all I can tell you ;-)
Where is this product change coming from?

Note that our requirements are automatically tested and this one is [here](https://github.com/DataDog/system-tests/blob/main/tests/appsec/test_traces.py#L49-L51).

Reply:
@Julio-Guerra let's sync on this, tomorrow.

Reply:
All clear :-)

Reply:
If we are going to do this this way, we should expose constants not use magic numbers.